### DISCUSSION TITLE
Making an accessible app

### DISCUSSION THREAD

**Discussion by annaecookux** [elementary/edw#22] [Accepted Talks]
With digital products becoming increasingly integral to our lives, how can designers and developers make sure our work is inclusive to disabled people? In this session, we will talk about how accessibility creates amazing products, applications, and experiences as well as the basic considerations of accessibility both in design and code. Learn the basics about accessibility laws, guidelines, and best practices so you can make your work even better!

**Comment by techdev5521** [comment]
I'd love to help with this if possible. As a visually impaired user who loves the way elemetnary is going, I'd really like to dig into this and potentially discuss best practices for Vala/Granite in particular and/or potentially improving on Granite's accessibility abilities.

**Reply by danirabbit** [reply]
@techdev5521 I wonder would you consider giving a whole separate talk about your experiences with elementary OS and AppCenter apps from an a11y perspective? I feel like there's probably a lot to talk about and I want to make sure we can give time to both your experiences so people can see the direct impact as well as a more instructional session

**Reply by techdev5521** [reply]
@danrabbit Just to see if I'm tracking, are you asking for a talk about what works and what doesn't for just using elementary as a daily driver for a visually impaired person?

**Reply by danirabbit** [reply]
@techdev5521 yeah if you would be interested, I think it would be good for people to see what that experience looks like

**Reply by techdev5521** [reply]
Yeah. My primary usage of elementaryOS these days is playing with elementaryOS 6 and doing simple dev. Perhaps using that use case and commenting on the accessibility of it would be a way to go.

**Comment by cassidyjames** [comment]
@annaecookux I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.

**Comment by annaecookux** [comment]
Hi all,

I did not see, that's great! I'll get started.

Thanks,
Anna
